src/transcript-plugin.tsx

```diff
@@ -1,401 +1,294 @@
+            kitchenSinkActive={!!this.sidePanelsManager.isItemActive(this._transcriptPanel)}
+            toggledWithEnter={this._triggeredByKeyboard}
+            isLive={isLive}
+            onClose={this.onClose}
+          />
+        );
+      },
+      iconComponent: ({isActive}: {isActive: boolean}) => {
+        return (
+          <Tooltip label={buttonLabel} type="bottom">
+            <PluginButton isActive={isActive} onClick={this.onClose} label={buttonLabel} />
+          </Tooltip>
+        );
       },
-      err => {
-        // remove getting captions from error while client fixed (https://kaltura.atlassian.net/browse/FEV-470)
-        this._getCaptionData(err, captionAsset);
-        // this._onError(err, "Failed to fetch caption asset", "_loadCaptionsAsset");
+      presets: [ReservedPresetNames.Playback, ReservedPresetNames.Live, ReservedPresetNames.Ads],
+      position: position,
+      expandMode: expandMode,
+      onActivate: () => {
+        this._pluginState = PluginStates.OPENED;
       }
-    )
-  };
+    });
 
-  private _getCaptionData = (data: any, captionAsset: KalturaCaptionAsset) => {
-    const rawCaptions = get(data, 'error.message', data);
-    if (rawCaptions) {
-      this._captions = this._parseCaptions(rawCaptions, captionAsset);
-      this._isLoading = false;
-      this._updateKitchenSink();
-    } else {
-      this._onError(undefined, "Captions data is empty", "_loadCaptionsAsset");
+    if ((expandOnFirstPlay && !this._pluginState) || this._pluginState === PluginStates.OPENED) {
+      this.ready.then(() => {
+        this.sidePanelsManager.activateItem(this._transcriptPanel);
+      });
     }
   }
 
-  private _parseCaptions = (data: string, captionAsset: KalturaCaptionAsset): CaptionItem[] => {
-    try {
-      const captionFormat = this._getCaptionFormat(captionAsset);
-      if (data.toString().indexOf("Error: ") === 0) {
-         // remove this condition once client fixed 
-        data = data.toString().replace("Error: ", "");
-      }
-      return getCaptionsByFormat(data, captionFormat);
-    } catch(err) {
-      this._onError(err, "Failed to parse the caption file", "_parseCaptions");
+  private _initLoading() {
+    if (!this._isLoading || this._hasError) {
+      this._isLoading = true;
+      this._hasError = false;
+      this._updateTranscriptPanel();
     }
-    return [];
-  };
-
-  private _getCaptionFormat = (captionAsset: KalturaCaptionAsset): string => {
-    const selectedLanguage: Record<string, any> =
-        this._captionsList &&
-        captionAsset &&
-        (this._captionsList.find((item: KalturaCaptionAsset) => item.id === captionAsset.id) || {});
-    return get(selectedLanguage, 'format', '');
-  };
+  }
 
   private _seekTo = (time: number) => {
-    this._player.currentTime = time;
+    this.player.currentTime = time;
   };
 
   private _handleDownload = () => {
-    const { playerConfig } = this._configs;
+    const {config} = this.player;
```

Comment:
use `this._activeCaptionMapId` to get array of captions from `this._captionMap` and then use them for download